frontend/js/tests/stats/UserEntityChart.test.tsx

```diff
@@ -28,14 +28,8 @@ const userProps = {
     id: 0,
     name: "dummyUser",
   },
-  apiUrl: "apiUrl",
-  newAlert: jest.fn(),
```

Comment:
apiUrl should stay there

Reply:
UserEntityChartProps does not include apiUrl neither is apiUrl being used anywhere in UserEntityChart. Do let me how to proceed with this.

Reply:
You're absolutely right! My bad, the props weren't typed properly so I didn't even see that.
Ignore this comment then :)